### ISSUE TITLE
Update the example page with more Gitpress specific content

### ISSUE THREAD

**Issue by pschoffer** [pschoffer/gitpress#1]
Mostly write some page based on the README in this repo. And make it as homepage + clear the clutter.

The change should be done in https://github.com/pschoffer/gitpress-example repo.

**Comment by forgondolin** [comment]
So, you want here a new wp page in the example repo with more information about GitPress features?

**Comment by pschoffer** [comment]
Yep exactly. I guess right now it just says Hello Word or something.

So ideally something like this:
* make a fork of the excample repo
* Start gitpress from your forked repo  (in FETCH mode)
* Change wordpress as you normally would. Putting up some Gitpress specific homepage and mabe getting rid of posts or something
* Then you might need to do something like `docker exec -it <docker_container> bash`, where you do `git push`
* And then finally PR from your forked repo to the gitpress-example master.

**Comment by forgondolin** [comment]
So What would you like to be written on the page. Something relevant to the project?

**Comment by pschoffer** [comment]
Yeah, like human-friendly README file alternative. I guess eventually it could be a base for the project's website if there would be one.
